code/game/machinery/suit_storage_unit.dm

```diff
@@ -380,6 +380,7 @@
 		choices,
 		custom_check = CALLBACK(src, PROC_REF(check_interactable), user),
 		require_near = !issilicon(user),
+		autopick_single_option = FALSE
```

Comment:
Why does this fix it? ``choices`` should have the ability to close and lock it, no? Why does ``show_radial_menu`` think there's only one?

Reply:
> should have the ability to open/disinfect/lock it, no? 

Yes that is true when it's not locked however when it does get locked it displays just 1 option

https://github.com/tgstation/tgstation/blob/1cafa900e630c38b577fbf540f81c5f0ae93c683/code/game/machinery/suit_storage_unit.dm#L360-L361

And the radial menu just auto picks this without giving the player a chance to select/unlock it